functions/src/modules/SQR.functions.ts

```diff
@@ -205,3 +206,87 @@ export const processSubmissions = functions.database.ref('/webforms/sqr/{submiss
     
     return 1;
 });
+
+
+/////////////////////////////////////////////////
+//          Update Email Templates on sendInBlue 
+//          in response to changes of the templates
+//          on FB (Storage Triggered)
+//
+/////////////////////////////////////////////////
+
+export const updateEmailTemplates = functions.storage.object()
+.onFinalize(async object => {
+
+    const path = require('path');
+    const os = require('os');
+    const fs = require('fs');
+
+    const filePath = object.name;
+    // slice(0, -5) to get rid of the trailing '.html' extension
+    const fileName = filePath.split('/')[2].slice(0, -5);
+    const tempLocalFile = path.join(os.tmpdir(), fileName);
+    
+    
+    if (!filePath.startsWith('email/templates')) 
```

Comment:
That is why I was asking about a separate bucket. As this function will be fired on every other files upload, which is not good. Please comment in the card.